extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -165,9 +165,12 @@ function () {
 			}
 		);
 
-		return $botIds;
+		return $botIds ?: [''];
```

Comment:
Returning empty array breaks the sql query the same way null does - it resolves to "WHERE NOT IN ()". Returning [''] gives us "WHERE NOT IN ('')", in the worst case scenario, which doesn't raise an error in staging env